apps/src/templates/JsDebugger.jsx

```diff
@@ -45,17 +45,17 @@ var DebugButtons = function () {
   return (
     <div id="debug-commands" className="debug-commands">
       <div id="debug-buttons">
-        <span> </span>
+        {" " /* Explicitly insert whitespace so that this behaves like our ejs file*/}
         <button id="pauseButton" className="debugger_button">
           <img src="/blockly/media/1x1.gif" className="pause-btn icon21"/>
           {i18n.pause()}
         </button>
-        <span> </span>
+        {" " /* Explicitly insert whitespace so that this behaves like our ejs file*/}
         <button id="continueButton" className="debugger_button">
```

Comment:
I appreciate preserving EJS behavior exactly, but I suspect the intended behavior here is pretty clear... which is why I question the space between `pauseButton` and `continueButton`.  Fine for now, but I'd feel free to rework these extra spaces whenever we manage to move the business logic into this component too.